examples/asr/emformer_rnnt/pipeline_demo.py

```diff
@@ -67,15 +67,15 @@ def run_eval_streaming(args):
                 features, length = streaming_feature_extractor(segment)
                 hypos, state = decoder.infer(features, length, 10, state=state, hypothesis=hypothesis)
             hypothesis = hypos[0]
-            transcript = token_processor(hypothesis.tokens, lstrip=False)
+            transcript = token_processor(hypothesis[0], lstrip=False)
```

Comment:
Does libtorch-light support dictionary? `hypothesis["tokens"]` is more intuitive to users than `hypothesis[0]`

Reply:
Dictionary should be used when keys can vary at runtime.
The structure of `Hypothesis` does not change at runtime, so dictionary is not appropriate.

Defining `token_index = 0` would be the best it can do.